File: GerenciaMusic360/ClientApp/app/shared/models/project-state.ts
import { IStatusProject } from "./status-project";
import { IUser } from "./user";
import { Audit } from "./audit";

export interface IProjectState extends Audit {
    id: number;
    statusProjectId: number;
    date: string;
    notes: string;
    statusRecordId: number;
    approverUserId: number;
    statusProject?: IStatusProject;
    userProfile: IUser;
}

File: GerenciaMusic360.Entities/ProjectState.cs
using System;
using System.ComponentModel.DataAnnotations.Schema;

namespace GerenciaMusic360.Entities
{
    public class ProjectState
    {
        public int Id { get; set; }
        public int StatusProjectId { get; set; }
        public DateTime Date { get; set; }
        public string Notes { get; set; }
        public long ApproverUserId { get; set; }
        public int ProjectId { get; set; }
        public DateTime Created { get; set; }
        public string Creator { get; set; }

        public virtual StatusProject StatusProject { get; set; }
        [ForeignKey("ApproverUserId")]
        public virtual UserProfile UserProfile { get; set; }
    }
}
